.github/workflows/test.yaml

```diff
@@ -34,57 +57,20 @@ jobs:
       # this script is hardcoded to build for linux/amd64
       - name: Prune docker cache
         run: docker system prune -f
-      - name: Build unit test image
-        run: python3 weave/docker/docker_build.py build_deps weave-test-python-client builder . weave/docker/Dockerfile.python-client.test
-      - name: Build integration test image
-        run: python3 weave/docker/docker_build.py build weave-integration-test . weave/legacy/Dockerfile.test
-
-  lint:
-    name: Python lint
-    timeout-minutes: 7
-    needs:
-      - build-container
-    # runs-on: [self-hosted, gke-runner]
-    runs-on: ubuntu-latest
-    container: us-east4-docker.pkg.dev/weave-support-367421/weave-images/weave-test-python-client:${{ github.sha }}
-    steps:
-      - uses: actions/checkout@v2
-      - run: source /root/venv/bin/activate && pre-commit run --hook-stage=pre-push --all-files
-
-  weavejs-lint-compile:
-    name: WeaveJS Lint and Compile
-    runs-on: ubuntu-latest
-    # runs-on: [self-hosted, gke-runner]
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-          token: ${{ secrets.GITHUB_TOKEN }}
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "18.x"
-      - run: |
-          set -e
-          cd weave-js
-          yarn install --frozen-lockfile
-          yarn generate
-          yarn eslint --max-warnings=0
-          yarn tslint
-          yarn prettier
-          yarn run tsc
+      - name: Build legacy (query sevice) unit test image
+        run: python3 weave/docker/docker_build.py build_deps weave-test-python-client builder . weave/legacy/Dockerfile.client.test
```

Comment:
again, this is not really about testing the `client` as much as it testing the query service. name seems misleading